src/gtest/test_checktransaction.cpp

```diff
@@ -109,10 +107,10 @@ void CreateJoinSplitSignature(CMutableTransaction& mtx, uint32_t consensusBranch
     }
 
     // Add the signature
-    assert(crypto_sign_detached(&mtx.joinSplitSig[0], NULL,
-                         dataToBeSigned.begin(), 32,
-                         joinSplitPrivKey
-                        ) == 0);
+    assert(rust_ed25519_sign(
```

Comment:
Do these functions need to have the `rust_` prefix? It's an implementation detail that they call into Rust. `tracing_*` doesn't have that prefix, and I think we will probably not want to use it in general.

Reply:
I added the `rust_` prefix for a bit of domain separation (because Ed25519 is more common than `tracing`), but I guess it's reasonable that we'd only expect to have e.g. a single `ed25519_sign` function in scope, so I'll remove the prefixes.